src/components/subsetDraft/Classification.jsx

```diff
@@ -154,38 +154,63 @@ export const Classification = ({item = {}, update, remove, from, to}) => {
 export const Codes = ({from, to, codes = [], id, include}) => {
     const {t} = useTranslation();
 
+    const codesToLoad = 35; // This is how many codes we load on first render. The rest can wait.
```

Comment:
This is great that you managed to find a solution. 👍 
**Pros:** it's clear and simple!
**Cons:** 
1. It does not depend on a screen it loads on. Why 35? The size of the screen has to affect amount of items the application loads.
2. Avoid hard coded values (magic numbers)!

Reply:
I've thought about this: Whether the initial amount of items loaded is 5 or 35 makes no difference to loading times. It feels instant regardless. 35 items is enough to fill beyond the bottom of a vertical computer screen. I figured that a vertical computer screen is the longest screen size anyone is going to use, and so 35 items will be more than enough to fill any other screen. Additional logic to calculate the exact number of items needed to fill the screen will only slow display, and make extra work for us that is not noticed by the end user experience in any way. In the outlier case where someone has a very very long screen that fits more than 35 units, it will only take 1-1.5 seconds to load the rest of the codes, to fill up their screen. Just having the 35 first load instantly helps making sure that the user know their click registered, and that they feel the site is fast and responsive. If the screen is very very long, filling the screen completely using additional logic to calculate screen size might give us the same, long load time that I am fixing with this solution. In other words: Screens that take more than 35 items to fill should probably not be filled on the first render.

Reply:
You've convinced me 😄 
We have remember to revise this number every time the style of the page is changed.